Sources/InstrumentationBaggage/Baggage.swift

```diff
@@ -99,7 +99,7 @@ extension Baggage {
     /// in order to inform other developers that the lack of baggage passing was not done on purpose, but rather because either
     /// not being sure where to obtain a baggage from, or other framework limitations -- e.g. the outer framework not being
     /// baggage aware just yet.
-    public static var topLevel: Baggage {
+    static var topLevel: Baggage {
```

Comment:
this is a bad rule in swift format, can we not do it?

Reply:
Agreed, I disabled it.